Schemas/validate_rsis.py

```
@@ -0,0 +1,164 @@
+    frame_width = meta_json["size"]["x"]
+    frame_height = meta_json["size"]["y"]
+
+    # Validate state PNGs.
+    # We only check they're the correct size and that they actually exist and load.
+    for state in meta_json["states"]:
+        state_name: str = state["name"]
+
+        png_name = os.path.join(rsi, f"{state_name}.png")
+        try:
+            image = Image.open(png_name)
+        except Exception as e:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)